assets/app/scripts/controllers/overview.js

```diff
@@ -119,6 +226,7 @@ angular.module('openshiftConsole')
     // Sets up subscription for deploymentConfigs, associates builds to triggers on deploymentConfigs
     watches.push(DataService.watch("deploymentConfigs", $scope, function(deploymentConfigs, action, deploymentConfig) {
       $scope.deploymentConfigs = deploymentConfigs.by("metadata.name");
+      $scope.deploymentConfigs[""] = null;
```

Comment:
if the view is no longer looping over `$scope.deploymentConfigs`, I don't think we need this